test/DurableOrchestrationClientBaseTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
```

Comment:
These new test files will need to copyright headers as well.

Reply:
I will add it. Thank you for catching that.